dfx.json

```diff
@@ -20,6 +20,14 @@
       "build": "demos/vc_issuer/build.sh",
       "post_install": "demos/vc_issuer/provision",
       "dependencies": [ "internet_identity" ]
+    },
+    "issuer_custom_origin": {
+      "type": "custom",
+      "candid": "demos/vc_issuer/vc_demo_issuer.did",
+      "wasm": "demos/vc_issuer/vc_demo_issuer_custom_origin.wasm.gz",
+      "build": "demos/vc_issuer/build.sh --custom-origin -o demos/vc_issuer/vc_demo_issuer_custom_origin.wasm.gz",
+      "post_install": "bash -c 'demos/vc_issuer/provision --issuer-frontend-hostname https://nice-issuer-custom-orig.com --issuer-canister issuer_custom_origin --issuer-derivation-origin https://$(dfx canister id test_app).icp0.io'",
```

Comment:
I've just read that `dfx` sets these environment variables: `CANISTER_ID_<CANISTER_NAME_UPPERCASE>`

could switch to that

Reply:
Yes!

I also asked about env vars in dfx and Max shared this https://internetcomputer.org/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-envars with me

Reply:
actually I'll wait a bit until new versions of dfx are a bit more widespread (mine doesn't support it for instance). Feel free to track this somewhere!